src/execution_plan/record.h

```diff
@@ -71,7 +71,8 @@ void Record_Merge
 void Record_TransferEntries
 (
 	Record *to,
-	Record from
+	Record from,
+	bool make_volatile
```

Comment:
I assume the transferred values are now owned by `from`
if that the case please rename to `transfer_ownership`

Reply:
I think the comment is a bit confusing here, since `from` still has ownership upon its entries. It's a copy mechanism, not a transfer.
I'll remove this part of the comment, I think it's unnecessary.